src/models.py

```diff
@@ -237,13 +237,15 @@ def build_embeddings(args, vocab, tasks, pretrained_embs=None):
             d_word = word_embs.size()[1]
             log.info("\tUsing pre-trained word embeddings: %s",
                      str(word_embs.size()))
+        elif args.word_embs=="scratch":
+            log.info("\tScratch Word embs!")
```

Comment:
Mind clarifying this (i.e., using a full sentence)?

Reply:
Fixed.